src/streaming/models/CmsdModel.js

```diff
@@ -212,7 +164,7 @@ function CmsdModel() {
         }
 
         // Get object type
-        let ot = OBJECT_TYPES.STREAM;
+        let ot = CmsdObjectType.STREAM;
```

Comment:
`CmsdObjectType.STREAM` seems to `undefined`. If we need `stream` in multiple places, please add a constant in this class. Ideally this can be added to the CommonMediaLibrary